docs/playbook-reference/defining-playbooks/playbook-basics.rst

```diff
@@ -88,6 +88,43 @@ In order to override the default configuration of the same playbook, both disabl
       - image_pull_backoff_reporter: {}
 
 
+Organizing Playbooks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
For the next two lines I suggest a slightly different explanation:

Using ``namedCustomPlaybooks``, you can define playbooks by name. This is useful when you want to define a base set of playbooks for all clusters/teams and then use additional Helm values files to override some of the base playbooks or add new ones.